calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,574 @@
+---
+description: Recommended Prometheus metrics for monitoring Calico Enterprise components.
+---
+
+# Recommended Prometheus metrics
+
+## Big picture
+
+Monitor the {{prodname}} Typha, Felix, and policy component metrics to ensure optimal cluster operation.
+
+## Concepts
+
+{{prodname}} Typha, Felix, and policy components are the most critical to monitor because they are responsible for ensuring networking and security functions are up-to-date and working as expected.
+
+### Typha
+
+Typha is a caching datastore proxy that sits between calico-nodes and Kubernetes API Server. Its primary function is to allow for increased cluster scale by reducing the load on Kubernetes API Server. Without Typha, large clusters (200+ nodes) would need a considerable amount of memory to correspond to the continuous watches and requests from calico-nodes running in the cluster.
+
+Typha maintains a single datastore connection on behalf of all of its clients (processes running in the calico-node pods, with Felix being Typhas’ main client). Typha watches for node, pod, network policy, bgp configuration, and other events on the Kubernetes API Server, caches and deduplicates this data, and fans out these events to its clients.
+
+### Felix
+
+Felix is a component of calico-node and is responsible for {{prodname}} network policy.
+Felix must be continuously in sync with the datastore to ensure the correct set of policies are applied to the node it is running on.
+
+![Typha-felix](/img/calico-enterprise/typha-felix.png)
+
+### About metrics
+
+Each {{prodname}} component that you want to connect to Prometheus for endpoint metrics has its own configuration (bgp, license, policy, felix, and typha).
+
+Note that Felix is a separate application with metric endpoints, its own core metrics to monitor itself, and a separate port for a second policy metric endpoint.
```

Comment:
I think we need a guide that explains how to connect to each of these metrics ports.  How is a user supposed to get from this doc to having a working prometheus set-up that is polling the correct ports?

Reply:
I've added docs backlog ticket.